lib/PuppeteerSharp/Frame.cs

```diff
@@ -74,10 +76,16 @@ internal Frame(FrameManager frameManager, CDPSession client, Frame parentFrame,
         }
 
         #region Properties
+        internal ICollection<Frame> _childFrames { get; } = new ConcurrentSet<Frame>();
+
         /// <summary>
         /// Gets the child frames of the this frame
         /// </summary>
-        public List<Frame> ChildFrames { get; } = new List<Frame>();
+        public List<Frame> ChildFrames {
+            get {
+                return _childFrames.OrderBy(frame => frame.Id).ToList();
```

Comment:
I think this won't work because. `ParentFrame.ChildFrames.Add(this);` will add child to a copy.

Reply:
Touche, I forgot to change that line